swift_browser_ui_frontend/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "swift_browser_ui_frontend_npm",
+  "version": "0.1.0",
```

Comment:
somewhat inconsistent with the release, we should keep the same version as the releases.

Reply:
Bumped :+1: